scipy/special/_special_ufuncs_docs.cpp

```diff
@@ -4927,6 +4963,42 @@ const char *mathieu_sem_doc = R"(
     --------
     mathieu_a, mathieu_b, mathieu_cem
 
+    Notes
+    -----
+    Odd Mathieu functions are the solutions to Mathieu's differential equation
+
+    .. math::
+
+        \frac{d^2y}{dx^2} + (a_m - 2q \cos(2x))y = 0
```

Comment:
```suggestion
        \frac{d^2y}{dx^2} + (b_m - 2q \cos(2x))y = 0
```
Maybe do this to be suggestive of `mathieu_b`?

Reply:
Feels like that might be confusing; we're not solving a different equation.

Reply:
Well, it's the same form of the equation but with a different infinite set of parameters that we call $b_m$ instead of $a_m, according to the name of the function.